CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added:
 
+- RS-xyz: Add record deletion functionality, [PR-xy](https://github.com/reductstore/web-console/pull/xy)
```

Comment:
We don't need RS-xxx ticket anymore because we don't use internal Jira anymore. The PR number and  url you know.